index.html

```diff
@@ -39,6 +40,221 @@ <h3>LET'S CONNECT</h3>
                     </ul>
                 </section>
         </div>
+         <!-- work -->
+         <div id="work_1">
+            <div class="project">
+                <img src="images/header/tonic.png" alt="tonic">
+            </div>
+            <h4 class="project_detail">Tonic</h4>
+            <div class="job_detail">
+                <h4 class="canopy">CANOPY</h4>
+                <p class="dot">•</p>
+                <p class="job">Back End Dev</p>
+                <p class="dot">•</p>
+                <p class="yrs">2015</p>
+            </div>
+            <div class="read">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="progm_skills">
+                <ul>
+                    <li class="skill_list1">
+                        <button>html</button>
+                    </li>
+                    <li class="skill_list2">
+                        <button>CSS</button>
+                    </li>
+                    <li class="skill_list3">
+                        <button>javaScript</button>
+                    </li>
+                </ul>
+            </div>
+            <a href="#" class="seeProj"><button>See Project</button></a>
+        </div>
+        <!-- 2nd  -->
+        <div id="work_2">
+            <div class="project">
+                <img src="images/header/second.png" alt="tonic">
+            </div>
+            <h4 class="project_detail">Multi-Post Stories</h4>
+            <div class="job_detail">
+                <h4 class="canopy">CANOPY</h4>
+                <p class="dot">•</p>
+                <p class="job">Back End Dev</p>
+                <p class="dot">•</p>
+                <p class="yrs">2015</p>
+            </div>
+            <div class="read">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="progm_skills">
+                <ul>
+                    <li class="skill_list1">
+                        <button>html</button>
+                    </li>
+                    <li class="skill_list2">
+                        <button>CSS</button>
+                    </li>
+                    <li class="skill_list3">
+                        <button>javaScript</button>
+                    </li>
```

Comment:
[OPTIONAL] These are list items and not buttons, so using ul and li tags to list them is ok. No need for button tags since they are not buttons